net-misc/onedrivegui/onedrivegui-1.1.1-r2.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+M_PN=OneDriveGUI
+
+DISTUTILS_SINGLE_IMPL=1
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=(python3_{9..13})
+
+inherit desktop distutils-r1
+if [[ ${PV} == "9999" ]]; then
+	EGIT_REPO_URI="https://github.com/bpozdena/${M_PN}.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/bpozdena/${M_PN}/archive/refs/tags/v${PV}.tar.gz -> ${PN}-${PV}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/${M_PN}-${PV}"
+fi
+
+DESCRIPTION="A simple GUI for OneDrive Linux client, with multi-account support."
+HOMEPAGE="https://github.com/bpozdena/OneDriveGUI"
+
+RDEPEND=">=net-misc/onedrive-2.5
+	!net-misc/onedrivegui-bin
+	dev-python/requests
+	dev-python/pyside[gui(+),webengine(+),widgets(+)]
```

Comment:
You're missing `pythong_gen_cond_dep`, see https://projects.gentoo.org/python/guide/single.html#dependencies